mitiq/interface/mitiq_cirq/cirq_utils.py

```diff
@@ -100,3 +104,29 @@ def execute_with_depolarizing_noise(
     rho = simulator.simulate(circuit).final_density_matrix
     expectation = np.real(np.trace(rho @ obs))
     return expectation
+
+
+def generate_inverse_confusion_matrix(
```

Comment:
This function simulates the experimental estimation of the confusion matrix which fine but a quite overkilling.
Given p0 and p1 there should be a fixed analytical solution for the confusion matrix and its inverse without the need to call `cirq.measure_confusion_matrix`.

Fine to keep the function as it is now in this PR, as long as we open a new issue to simplify this function in a future step.

Reply:
Added an issue as #1562